platform/platform-configuration-listener/src/test/java/org/codice/ddf/configuration/store/ConfigurationFileDirectoryTest.java

```diff
@@ -82,11 +91,12 @@
         public final Map<Path, Path> filesMoved = new HashMap<>();
 
         public ConfigurationFileDirectoryUnderTest(
-                @NotNull DirectoryStream<Path> configurationDirectory, Path processedDirectory,
+                @NotNull DirectoryStream<Path> configurationDirectory,
```

Comment:
Just noticed that I put `@NotNull` on those parameters by mistake. Can you please remove them if you happen to touch this file?